bin/postgres-ha/modules/crunchyadm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+username="crunchyadm"
+
+# Create the crunchyadm user if it doesn't already exist
+if [[ "${PGHA_CRUNCHYADM}" == "true" ]] &&
+    [[ $(psql -v "username=${username}" -tA -f - <<< "SELECT 1 FROM pg_roles WHERE rolname=:'username'") != 1 ]]
+then
+    echo_info "Creating user ${username}"
+    psql -v "username=${username}" -tA -f - <<< "CREATE ROLE :username LOGIN"
```

Comment:
`:username` isn't being escaped here. Suggest:

```suggestion
    psql -v "username=${username}" -tA -f - <<< 'CREATE ROLE :"username" LOGIN'
```